app/lib/shared/utils.dart

```diff
@@ -10,9 +10,13 @@ import 'dart:io';
 import 'dart:math';
 
 import 'package:logging/logging.dart';
+import 'package:http/http.dart';
```

Comment:
Please prefix usages of `package:http` with `http`, it makes it so much easier to read, because `Client` could be anything, whereas `http.Client` read much more clearly.